thumbor/engines/pil.py

```diff
@@ -16,6 +16,16 @@
 
 from PIL import Image, ImageFile, ImageDraw, ImageSequence, JpegImagePlugin
 
+try:
```

Comment:
Why do we import cv2 if we never use it here?

Reply:
done!